components/button/button-add.js

```diff
@@ -69,6 +83,34 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 			:host([mode="icon-when-interacted"]) button:not(:focus):not(:hover) d2l-button-add-icon-text {
 				position: absolute;
 			}
+			:host([mode="icon-when-interacted"]) button:hover d2l-button-add-icon-text,
+			:host([mode="icon-when-interacted"]) button:focus d2l-button-add-icon-text {
+				animation: position-change-animation 50ms step-end 0ms 1 forwards; /* add delay in changing position to avoid flash of missing icon space */
+			}
+
+			@media (prefers-reduced-motion: reduce) {
+				button:hover .line,
+				button:focus .line {
+					animation: none !important;
+					background: var(--d2l-color-celestine-minus-1);
+				}
+				:host([mode="icon-when-interacted"]) button:hover .line {
+					transition: background 100ms step-end;
```

Comment:
When `prefers-reduced-motion` is enabled, we don't want the line animation but we do want a bit of a delay in changing the line colour to avoid flicker when user is moving their mouse around a page.